third_party/blink/renderer/modules/cobalt/h5vcc_system/h_5_vcc_system.cc

```diff
@@ -63,6 +65,42 @@ void H5vccSystem::OnGetLimitAdTracking(ScriptPromiseResolver* resolver,
   resolver->Resolve(result);
 }
 
+ScriptPromise H5vccSystem::concealOrStop(
+    ScriptState* script_state,
+    const ConcealOrStopCallbacks* callbacks,
+    ExceptionState& exception_state) {
+  auto* resolver = MakeGarbageCollected<ScriptPromiseResolver>(
+      script_state, exception_state.GetContext());
+  EnsureReceiverIsBound();
+
+  remote_h5vcc_system_->GetUserOnExitStrategy(
+      WTF::BindOnce(&H5vccSystem::OnGetUserOnExitStrategy, WrapPersistent(this),
+                    WrapPersistent(resolver), WrapPersistent(callbacks)));
+  return resolver->Promise();
+}
+
+ScriptPromise H5vccSystem::concealOrStop(ScriptState* script_state,
+                                         ExceptionState& exception_state) {
+  return concealOrStop(script_state, /*callbacks=*/nullptr, exception_state);
+}
+
+void H5vccSystem::OnGetUserOnExitStrategy(
+    ScriptPromiseResolver* resolver,
+    const ConcealOrStopCallbacks* callbacks,
+    h5vcc_system::mojom::blink::UserOnExitStrategy result) {
+  if (result == h5vcc_system::mojom::blink::UserOnExitStrategy::kStop &&
+      callbacks && callbacks->hasBeforeStop()) {
+    callbacks->beforeStop()->InvokeAndReportException(this);
+  }
+  if (result == h5vcc_system::mojom::blink::UserOnExitStrategy::kConceal &&
```

Comment:
tiny nit: I think an `else if` here makes it clearer to the reader that at most one of these will ever get executed in a call